source/module_hamilt_general/module_xc/test/CMakeLists.txt

```diff
@@ -290,6 +290,10 @@ AddTest(
     ../xc_funct_corr_gga.cpp ../xc_funct_corr_lda.cpp ../xc_funct_exch_gga.cpp
     ../xc_funct_exch_lda.cpp ../xc_funct_hcth.cpp
     ../../../module_base/matrix.cpp ../../../module_base/timer.cpp
+    ../../../module_base/formatter_fmt.cpp
```

Comment:
Maybe you can add a new library for fmt in CMakeLists.txt, rather than copy them in UTs.

Reply:
thanks, I have packed source files up and update my PR